tasks/release.py

```diff
@@ -462,46 +493,41 @@ def _query_github_api(auth_token, url):
     return response
 
 
-def _confirm_external_repo_version(repo, latest_version, highest_release_json_version):
-    confirmed_version = highest_release_json_version
-    if latest_version != highest_release_json_version:
-        print(
-            color_message(
-                "The latest version of {} ({}) does not match the latest version found in the release.json file ({}).".format(
-                    repo, _stringify_version(latest_version), _stringify_version(highest_release_json_version)
-                ),
-                "orange",
-            )
-        )
-        if yes_no_question(
-            "Do you want to update {} to {}?".format(repo, _stringify_version(latest_version)), "orange", False
-        ):
-            confirmed_version = latest_version
+##
+## Base functions to fetch candidate versions on other repositories
+##
 
-    return confirmed_version
 
+def _get_highest_repo_version(auth, repo, version_prefix, allowed_major_versions, version_re):
+    if not allowed_major_versions:
+        allowed_major_versions = [""]
```

Comment:
I don't understand why we do `[""]` instead of `[]`

Reply:
If you don't specify `allowed_major_versions`, that means you want to search for any major version. Setting to `[]` would mean not doing any iteration in the `for` loop below, which isn't what we want (we wouldn't find any version because we wouldn't be querying any version). Setting it to `[""]` allows to search all versions. I'll add a comment to explain that.